README.rst

```diff
@@ -66,30 +65,33 @@ Here is a slightly fancier example:
 
    >>> class fancy(Decorum):
    ...     def init(self, arg=None):
+   ...         super(fancy, self).init()
    ...         self.arg = arg
    ...
    ...     def wraps(self, f):
    ...         if self.arg:
-   ...             def newf():
-   ...                 print(self.arg)
+   ...             newf = lambda: self.arg
```

Comment:
I also uses `return` instead of `print`, so that I can easily intercept and post-process the result.